taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+          <a href="#cyberduck_transfer">Transfer Data with Cyberduck</a>
+          guide below.
+        </p>
+      </blockquote>
+      <p>
+        For users that are new to the command line, using either <code>scp</code> or <code>sftp</code> to transfer data is advised.
+      </p>
+      <h4 id="data_transfer_prereqs">
+        Prerequisites for Data Transfer with Command Line Tools
+      </h4>
+
+      <p>Before we begin, you will need to know:</p>
+
+      <ul>
+        <li>the path to your data file(s) on your local system</li>
+        <li>the path to your transfer directory on the remote storage server</li>
+      </ul>
+
+      <h4 id="local_data_path">
+        Determining the Path to Your Data File(s) on Your Local System
+      </h4>
+
+      <p>
+        In order to transfer your project data, you will need to know where your file(s) is located on your local system. To do so, navigate through the file system on your local computer.
+      </p>
+      <p>
+        For example, you can do so on Mac by going to Finder. Common data file locations such as Desktop and Documents should be accessible through the Favorites pane on the left-hand side of the window. You can also do Finder &gt; Go to find other common locations on your computer.
+      </p>
+      <p>
+        Once you have navigated to where your file(s) is located, take note of its path by right-clicking on the file and selecting Get Info.
+      </p>
+      <figure id="figure1">
+        <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/6c/ea/6ceacba9-7824-4925-b924-dd317040b629/picture1.png__1170x0_q85_subsampling-2_upscale.png" />
+        <figcaption>
+          Figure 1. Use Get Info to determine “Where” the path of your data file(s) is
+        </figcaption>
+      </figure>
+
+      <p>
+        For example, a file located in an <code>portal-data</code> folder in
+        <code>Documents</code> would have the following path:
+      </p>
+      <p>
+        <code>/Users/myname/Documents/portal-data/my_file.txt</code>
+      </p>
+      <h4 id="storage_data_path">
+        Determining the Path to Your Transfer Directory
+      </h4>
+
+      <p>
+        A transfer directory on the remote storage server has already been set up for you and is available for data transfer. The transfer directory path is individualized to your research center/institution.
+      </p>
+      <details class="small"><summary>Examples:</summary>
+        <pre><code>/corral-secure/projects/A2CPS/submissions/utaustin/</code></pre>
+        <blockquote>
+          <p>
+            Note: The transfer directory name will always be one word and in lowercase.
+          </p>
+        </blockquote>
+      </details>
+      <p>
+        If you are unsure of your transfer directory path, please consult the email you have received from a relevant contact <samp>@tacc.utexas.edu</samp>.
```

Comment:
The "relevant contact" is mentioned repeatedly, how would the user determine that information?

Reply:
I do not know.

The text originally was, as in [A2CPS's Rsync doc](https://a2cps-staging.tacc.utexas.edu/help/rsync-data-transfer-guide/):
> If you are unsure of your transfer directory path, please consult the email you have received from [acantu@tacc.utexas.edu](mailto:acantu@tacc.utexas.edu), or contact [acantu@tacc.utexas.edu](mailto:acantu@tacc.utexas.edu) directly.

Reply:
I would recommend the following:

"If you are unsure of your transfer directory path, please consult your project PI directly."

If they do not know who their PI is, that is an entirely different class of problem.